quarkchain/cluster/jsonrpc.py

```diff
@@ -1280,6 +1280,8 @@ async def setTargetBlockTime(self, root_block_time=0, minor_block_time=0):
     @decode_arg("address", address_decoder)
     @decode_arg("limit", quantity_decoder)
     async def getTotalBalance(self, block_hash, token_id, address, limit="0x64"):
+        if limit > 10000:
```

Comment:
how about following `getTransactionsByAddress`
![image](https://user-images.githubusercontent.com/38544039/87482029-76d3dc00-c5e5-11ea-8de7-5a9bb6aa9017.png)

Reply:
Well, if we change the limit silently, the user cannot know how many accounts did the returned balances stand for, since we didn't return the changed limit.

Reply:
but it's a `limit`, not a `size` parameter. we don't have the API contract that says the returned balance is guaranteed to be for that many addresses, but only for pagination purpose

Reply:
Make sense